Makefile

```diff
@@ -0,0 +1,24 @@
+SKIP_SQUASH?=0
+VERSIONS="2.4.41 2.4.41testenv"
+
+ifeq ($(TARGET),rhel7)
+	OS := rhel7
+else
+	OS := centos7
+endif
+
+ifeq ($(VERSION), 2.4.41)
+	VERSION := 2.4.41
+else ifeq ($(VERSION), 2.4.41testenv)
+    VERSION := 2.4.41testenv
+else
+	VERSION :=
+endif
+
+.PHONY: build
+build:
+	SKIP_SQUASH=$(SKIP_SQUASH) VERSIONS=$(VERSIONS) hack/build.sh $(OS) $(VERSION)
+
+.PHONY: test
+test:
+	SKIP_SQUASH=$(SKIP_SQUASH) VERSIONS=$(VERSIONS) TEST_MODE=true hack/build.sh $(OS) $(VERSION)
```

Comment:
there have been some recent updates here, please diff w/ a current Makefile from one of the other image repos

same on the test/run script